tests/index.html

```diff
@@ -11,7 +11,7 @@
     {{content-for "test-head"}}
 
     <link rel="stylesheet" href="{{rootURL}}assets/vendor.css">
-    <link rel="stylesheet" href="{{rootURL}}assets/gsoc-prep-tasks.css">
+    <link rel="stylesheet" href="{{rootURL}}assets/git-task-list.css">
```

Comment:
Can we use `modulePrefix` instead of hardcoding `git-task-list`, or maybe use a common name like `app` / `main` ?

Reply:
@bekicot said it's not possible.

Reply:
Well, when I use `{{modulePrefix}}`, it literally use `{{modulePrefix}}`